api/src/main/java/org/pmiops/workbench/cohorts/CohortService.java

```diff
@@ -68,11 +89,36 @@ public CohortService(
         .build();
   }
 
-  public TerraJob submitGenomicsCohortExtractionJob(String workspaceNamespace, String workspaceName)
+  private List<String> getPersonIds(Long cohortId) {
+    String cohortDefinition = cohortDao.findOne(cohortId).getCriteria();
+
+    final SearchRequest searchRequest = new Gson().fromJson(cohortDefinition, SearchRequest.class);
+
+    final QueryJobConfiguration participantIdQuery =
+        cohortQueryBuilder.buildParticipantIdQuery(new ParticipantCriteria(searchRequest));
+
+    return Streams.stream(
+            bigQueryService
+                .executeQuery(bigQueryService.filterBigQueryConfig(participantIdQuery))
+                .getValues())
+        .map(personId -> personId.get(0).getValue().toString())
+        .collect(Collectors.toList());
+  }
+
+  public TerraJob submitGenomicsCohortExtractionJob(DbWorkspace workspace, Long cohortId)
       throws ApiException {
     WorkbenchConfig.WgsCohortExtractionConfig cohortExtractionConfig =
         workbenchConfigProvider.get().wgsCohortExtraction;
 
+    FirecloudWorkspace fcWorkspace = fireCloudService.getWorkspace(workspace).get().getWorkspace();
+    String extractionUuid = UUID.randomUUID().toString();
+    String filename = extractionUuid + "_person_ids.txt";
```

Comment:
We will need a product decision here, but I don't think we want to be writing this into the root of their bucket. I would put all of this stuff under a prefix; and within that I would put a cohort extraction identifier. Maybe something like


```
gs://bkt
  /cohort-variants
    /<extract_id (maybe include date here)>
      /...
```